src/main/java/useCases/createCorporatePage/PageGeneratePresenter.java

```diff
@@ -0,0 +1,14 @@
+package useCases.createCorporatePage;
+
+public class PageGeneratePresenter {
```

Comment:
Should this implement the output boundary? Also since our front-end is not in Java, we might not use a presenter. When the user makes a request from the front-end, the controller will take in the request and result in a response. We will then return that response to the front-end.